packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1453,9 +1454,7 @@
   "invisible": "invisible",
   "Invisible": "Invisible",
   "Invitation": "Invitation",
-  "Invitation_HTML": "Invitation HTML",
-  "Invitation_HTML_Default": "<h1>You have been invited to <strong>[Site_Name]</strong></h1><p>Go to [Site_URL] and try the best open source chat solution available today!</p>",
-  "Invitation_HTML_Description": "You may use the following placeholders: <br/><ul><li>[email] for the recipient email.</li><li>[Site_Name] and [Site_URL] for the Application Name and URL respectively.</li></ul>",
+  "Invitation_Email_Description": "You may use the following placeholders: <br/><ul><li>[email] for the recipient email.</li><li>[Site_Name] and [Site_URL] for the Application Name and URL respectively.</li></ul>",
```

Comment:
get you _a_ new one

Reply:
Did this maybe end up on the wrong line?